downloads.html

```diff
@@ -22,13 +43,13 @@ <h3 class="download-version">1.1.0<a href="#1.1.0"><i class="fas fa-link " style
                 <a href="https://www.apache.org/dist/kafka/1.1.0/RELEASE_NOTES.html">Release Notes</a>
             </li>
             <li>
-                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/1.1.0/kafka-1.1.0-src.tgz">kafka-1.1.0-src.tgz</a> (<a href="https://www.apache.org/dist/kafka/1.1.0/kafka-1.1.0-src.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/1.1.0/kafka-1.1.0-src.tgz.sha512">sha512</a>)
+                Source download: <a href="https://archive.apache.org/dist/kafka/1.1.0/kafka-1.1.0-src.tgz">kafka-1.1.0-src.tgz</a> (<a href="https://archive.apache.org/dist/kafka/1.1.0/kafka-1.1.0-src.tgz.asc">asc</a>, <a href="https://archive.apache.org/dist/kafka/1.1.0/kafka-1.1.0-src.tgz.sha512">sha512</a>)
```

Comment:
Are these right? I think we've been told we're not supposed to use the dist URLs and instead use the mirrors.

Reply:
This link was copied and modified based on the download link for 1.0.0 source code which in turn is from https://github.com/apache/kafka-site/pull/148/files.

I am not very sure what is the mirror URL and whether we should use that. It seems like an existing issue here. If we can have quick answer (e.g. a download link to be used here) then I am happy to update it.

Reply:
https://issues.apache.org/jira/browse/KAFKA-6222 says that we should use https://www.apache.org/dist/kafka but we should not use dist.apache.org. So it looks like the current download links are OK?

Reply:
ah, ok, i was just misremembering the exact URLs